snyk_eclipse_plugin/src/io/snyk/eclipse/plugin/runner/ProcessRunner.java

```diff
@@ -54,9 +65,19 @@ public ProcessBuilder createMacProcessBuilder(String command, Optional<String> p
 	}
 	
 	public ProcessBuilder createWinProcessBuilder(String command, Optional<String> path) {
-		if (command.startsWith("/")) command = command.replaceFirst("/", "");
-		ProcessBuilder pb = new ProcessBuilder("cmd.exe", "/c", command);
+		if (command.startsWith("\"/")) command = command.replaceFirst("/", "");
+		ProcessBuilder pb = new ProcessBuilder("cmd.exe", "/c ", command);
```

Comment:
Should the space be there?

Reply:
no, `.startsWith` is used to remove first slash from path to the binary, e.g. `"/C:/Program Files...` => `"C:/Program Files...`